src/models/db.js

```diff
@@ -82,6 +96,23 @@ export default config => {
     if ('associate' in db[modelName]) {
       db[modelName].associate(db);
     }
+
+    const addDbHooks = () => {
```

Comment:
You don't need to define a function for each iteration of the loop. Just exec its content directly.